src/components/common/ui/alert-button-mui/AlertButton.styles.ts

```diff
@@ -0,0 +1,111 @@
+import { SxProps, Theme } from '@mui/material/styles';
+export const button = (variant, startIcon, endIcon, text): SxProps<Theme> => ({
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+  borderRadius: '4px',
+  color: 'gray.600',
+  textTransform: 'none',
+  fontSize: '14px',
+  fontWeight: '700',
+  minWidth: 'unset',
+  minHeight: 'unset',
+
+  ...((startIcon || endIcon) &&
+    text && {
+      gap: '6px',
+      padding: '8px 12px',
+    }),
+  ...((startIcon || endIcon) &&
+    !text && {
+      padding: '6px',
+      gap: 'none',
+    }),
+  ...(!(startIcon || endIcon) &&
+    text && {
+      padding: '8px 16px',
+      gap: '0',
+    }),
+
+  ...(variant == 'success' && {
+    backgroundColor: 'success.200',
+    border: `2px transparent solid`,
```

Comment:
looks like border is similar for all variants and states, there is just difference in colors. move it up to common styles